freegs/machine.py

```diff
@@ -98,32 +98,6 @@ def __getitem__(self, name):
             if label == name:
                 return coil
         raise KeyError("Circuit does not contain coil with label '{0}'".format(name))
-    
```

Comment:
Are these getters and setters not needed on circuits?

Reply:
Ooops — sorry — that commit shouldn't have been there. I've removed this now, thanks!